packages/vaadin-combo-box/src/vaadin-combo-box-mixin.js

```diff
@@ -1021,8 +1021,8 @@ export const ComboBoxMixin = (subclass) =>
       if (this._toggleElement) {
         this._toggleElement.addEventListener('click', this._preventDefault);
       }
-      if (this._clearElement) {
-        this._clearElement.addEventListener('click', this._preventDefault);
+      if (this.clearElement) {
+        this.clearElement.addEventListener('click', this._preventDefault);
```

Comment:
I'm not sure if this code is needed, tests passed without it 🤷‍♂️

Reply:
Actually there is a regression, I will submit a separate PR to avoid input blur on clear button mousedown.